README.md

```diff
@@ -323,6 +323,17 @@ If your beta feature requires a `Stripe-Version` header to be sent, use the `Str
 Stripe.api_version += "; feature_beta=v3"
 ```
 
+### Custom requests
+
+If you would like to send a request to an undocumented API (for example you are in a private beta), or if you prefer to bypass the method definitions in the library and specify your request details directly, you can use the `raw_request` method on `Stripe`.
+
+```ruby
+resp = Stripe.raw_request(:post, "/v1/beta_endpoint", {param: 123}, {stripe_version: "2022-11-15; feature_beta=v3"})
+
+# (Optional) resp is a StripeResponse. You can use `Stripe.deserialize` to get a StripeObject.
+deserialized_resp = Stripe.deserialize(resp)
```

Comment:
I think this actually needs to be `Stripe.deserialize(resp.http_body)` based on my testing. I recall saying we wanted to accept a string / return a string for consistency with the deserialize method